docs/reference/modules/ui.md

```diff
@@ -0,0 +1,221 @@
+
+ApostropheCMS uses CSS variables for consistent styling across the admin UI. These variables are defined in the `modules/@apostrophecms/ui/ui/apos/scss/global/_theme.scss` file.
+
+#### Colors
+
+```css
+--a-primary: #6516dd;           /* Default primary color */
+--a-danger: #eb443b;            /* Error/danger color */
+--a-success: #00bf9a;           /* Success color */
+--a-warning: #ffce00;           /* Warning color */
+--a-text-primary: #000;         /* Primary text color */
+--a-text-inverted: #fff;        /* Inverted text color */
+--a-background-primary: #fff;   /* Primary background color */
+```
+
+#### Typography
+
+```css
+--a-family-default: -apple-system, system-ui, "BlinkMacSystemFont", "Segoe UI",
+  "Roboto", "Helvetica", "Arial", sans-serif;
+--a-type-base: 12px;
+--a-type-label: 13px;
+--a-type-large: 14px;
+--a-type-heading: 22px;
+```
+
+#### Layout
+
+```css
+--a-border-radius: 5px;
+--a-border-radius-large: 10px;
+--a-box-shadow: 0 3px 13px 4px rgb(0 0 0 / 8%);
+```
+
+### Dark Mode Theme
+
+ApostropheCMS has a built-in dark mode theme that can be activated by adding the class `apos-theme-dark` to either the body element or the admin-bar wrapper, as [detailed below](#creating-custom-themes).
+
+To activate dark mode through the standard ApostropheCMS Nunjucks layout, use the `bodyClass` block in your template:
+
+<AposCodeBlock>
+
+```nunjucks
+{% extends "layout.html" %}
+
+{% block bodyClass %}apos-theme-dark{% endblock %}
+```
+  <template v-slot:caption>
+    views/layout.html
+  </template>
+</AposCodeBlock>
+
+::: tip
+If you need to add dark mode while keeping any existing body classes added to this same block use:
+
+```nunjucks
+{% block bodyClass %}{{ super() }} apos-theme-dark{% endblock %}
+```
+:::
+
+You can implement a front-end UI toggle between the default light theme and the dark theme by adding and removing the `apos-theme-dark` class.
+
+### Color Themes
+
+ApostropheCMS supports several pre-defined color themes that can be applied to the admin UI. These themes change the primary color used throughout the interface.
+
+Available themes:
+Available themes:
+- `apos-theme--primary-default` (Purple: <span class="theme-swatch" style="background:#6516dd"></span> #6516dd)
+- `apos-theme--primary-blue` (Blue: <span class="theme-swatch" style="background:#0062ff"></span> #0062ff)
+- `apos-theme--primary-orange` (Orange: <span class="theme-swatch" style="background:#e65100"></span> #e65100)
+- `apos-theme--primary-aqua` (Aqua: <span class="theme-swatch" style="background:#1eacc7"></span> #1eacc7)
+- `apos-theme--primary-sun` (Yellow: <span class="theme-swatch" style="background:#f7a704"></span> #f7a704)
+- `apos-theme--primary-green` (Green: <span class="theme-swatch" style="background:#00b075"></span> #00b075)
+- `apos-theme--primary-pink` (Pink: <span class="theme-swatch" style="background:#fd9ec0"></span> #fd9ec0)
+
+Unlike the dark theme, these classes need to be added to the admin-bar wrapper.
```

Comment:
Does that mean they are specific to only certain parts of the UI on the page? For instance, does this affect the admin UI of widgets on the page? How about modals?

Reply:
No - they apply to all the controls, but the function that adds the correct class onto the page will override the class being added to the body tag.